Lib/test/test_embed.py

```diff
@@ -0,0 +1,202 @@
+# Run the tests in Programs/_testembed.c (tests for the CPython embedding APIs)
+from test import support
+import unittest
+
+from collections import namedtuple
+import os
+import re
+import subprocess
+import sys
+
```

Comment:
Would you mind to add an empty line for PEP8, please? :-)

Reply:
Added!